integration/specs/Picklist/picklist-1.spec.js

```diff
@@ -11,10 +11,10 @@ const {
 } = require('../../constants');
 
 const PICKLIST = '#picklist-1';
+const PICKLISTMENU = 'div[role="listbox"]';
```

Comment:
We can't do this since this is implementation details of the InternalDropdown and is a fragile selector too since the page can have more elements with this selector, then this should be part of the picklist page object, I think we can use the id passed to the InternalDropdow to make this selector unique, if you look at the picklist there is a unique id generated `this.listboxId` that is passed as id to the InternalDropdown and as `aria-controls` to the input, then maybe you can get this `aria-controls` value and init an instance of the internal dropdown page object with this value.
Note that the page object of the InternalDropdown needs to be created and discussed so you need to come with a proposal of this page object